dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/InDevelopmentPavedRoadFramework.kt

```diff
@@ -14,15 +14,20 @@ import java.io.File
  * for any frameworks that are currently in development. It turns a large amount of errors into warnings
  * that make the development experience more pleasant
  */
+@Suppress("TooManyFunctions", "LongParameterList")
 abstract class InDevelopmentPavedRoadFramework(
     identifier: String,
     label: String,
     explanation: String,
     frameworkTemplateCsvFile: File,
     order: Int,
     enabledFeatures: Set<FrameworkGenerationFeatures> = FrameworkGenerationFeatures.ENTRY_SET,
+    privateFrameworkBoolean: Boolean = false,
```

Comment:
Vorschlag: vllt "isPrivateFramework" nennen. Dann ist es konsistenter mit den Namen von vielen anderen Booleans in unserem Code.